### ISSUE TITLE
Uyuni 2023.04 Notification Messages Server error, please check log files.

### ISSUE THREAD

**Issue by j-kihet** [uyuni-project/uyuni#6918]
### Problem description

When trying to see notifications an error is shown.
`Server error, please check log files.`

WebUI stop responding after a while.
Some Recurring States wont also finish. Seems to be states that run in to errors.
System list - pages  are also blank only showing sidebar.
Api also does not work but not sure if it is related to the issue.
### Steps to reproduce

1. Upgrade to 2023.04
2. Click on notifications

### Uyuni version

```bash
Information for package Uyuni-Server-release:
---------------------------------------------
Repository     : Uyuni Server Stable
Name           : Uyuni-Server-release
Version        : 2023.04-220400.204.2.uyuni2
Arch           : x86_64
Vendor         : obs://build.opensuse.org/systemsmanagement:Uyuni
Support Level  : Level 3
Installed Size : 1.4 KiB
Installed      : Yes
Status         : up-to-date
Source package : Uyuni-Server-release-2023.04-220400.204.2.uyuni2.src
Summary        : Uyuni Server
```

### Uyuni proxy version (if used)

```bash
Not used
```

### Useful logs

```bash
Output of /var/log/rhn/rhn_web_ui.log
 [ajp-nio-127.0.0.1-8009-exec-1] ERROR com.redhat.rhn.frontend.servlets.SessionFilter - Error during transaction. Rolling back
javax.servlet.ServletException: Filter execution threw an exception
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:196) ~[catalina.jar:9.0.43]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[catalina.jar:9.0.43]
	at com.redhat.rhn.frontend.servlets.AuthFilter.doFilter(AuthFilter.java:99) ~[rhn.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) ~[catalina.jar:9.0.43]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[catalina.jar:9.0.43]
	at com.opensymphony.module.sitemesh.filter.PageFilter.parsePage(PageFilter.java:142) ~[sitemesh.jar:?]
	at com.opensymphony.module.sitemesh.filter.PageFilter.doFilter(PageFilter.java:58) ~[sitemesh.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) ~[catalina.jar:9.0.43]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[catalina.jar:9.0.43]
	at com.redhat.rhn.frontend.servlets.LocalizedEnvironmentFilter.doFilter(LocalizedEnvironmentFilter.java:71) ~[rhn.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) ~[catalina.jar:9.0.43]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[catalina.jar:9.0.43]
	at com.redhat.rhn.frontend.servlets.EnvironmentFilter.doFilter(EnvironmentFilter.java:102) ~[rhn.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) ~[catalina.jar:9.0.43]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[catalina.jar:9.0.43]
	at com.redhat.rhn.frontend.servlets.SessionFilter.doFilter(SessionFilter.java:59) ~[rhn.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) ~[catalina.jar:9.0.43]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[catalina.jar:9.0.43]
	at com.redhat.rhn.frontend.servlets.SetCharacterEncodingFilter.doFilter(SetCharacterEncodingFilter.java:98) ~[rhn.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) ~[catalina.jar:9.0.43]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[catalina.jar:9.0.43]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202) ~[catalina.jar:9.0.43]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97) ~[catalina.jar:9.0.43]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542) ~[catalina.jar:9.0.43]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143) ~[catalina.jar:9.0.43]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92) ~[catalina.jar:9.0.43]
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687) ~[catalina.jar:9.0.43]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78) ~[catalina.jar:9.0.43]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346) ~[catalina.jar:9.0.43]
	at org.apache.coyote.ajp.AjpProcessor.service(AjpProcessor.java:433) ~[tomcat-coyote.jar:9.0.43]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65) ~[tomcat-coyote.jar:9.0.43]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:888) ~[tomcat-coyote.jar:9.0.43]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684) ~[tomcat-coyote.jar:9.0.43]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) ~[tomcat-coyote.jar:9.0.43]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) ~[tomcat-util.jar:9.0.43]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
Caused by: java.lang.NoClassDefFoundError: Could not initialize class com.redhat.rhn.domain.notification.UserNotificationFactory
	at com.suse.manager.webui.controllers.NotificationMessageController.dataUnread(NotificationMessageController.java:122) ~[rhn.jar:?]
	at com.suse.manager.webui.utils.SparkApplicationHelper.lambda$withUser$1(SparkApplicationHelper.java:98) ~[rhn.jar:?]
	at com.suse.manager.webui.utils.SparkApplicationHelper.lambda$asJson$10(SparkApplicationHelper.java:404) ~[rhn.jar:?]
	at spark.RouteImpl$1.handle(RouteImpl.java:72) ~[spark-corespark-core.jar:?]
	at spark.http.matching.Routes.execute(Routes.java:61) ~[spark-corespark-core.jar:?]
	at spark.http.matching.MatcherFilter.doFilter(MatcherFilter.java:134) ~[spark-corespark-core.jar:?]
	at spark.servlet.SparkFilter.doFilter(SparkFilter.java:174) ~[spark-corespark-core.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) ~[catalina.jar:9.0.43]
	... 37 more
```

### Additional information
@mcalmer 
Previous report https://github.com/uyuni-project/uyuni/issues/6552 
`ls -la /srv/tomcat/webapps/rhn/WEB-INF/lib/`
```
tomcat tomcat 12288 Apr 24 13:19 .
tomcat tomcat  4096 Apr 24 13:19 ..
tomcat tomcat    25 Apr 19 19:34 antlr.jar -> /usr/share/java/antlr.jar
tomcat tomcat    43 Apr 19 19:34 apache-commons-compress.jar -> /usr/share/java/apache-commons-compress.jar
tomcat tomcat    52 Apr 19 19:34 apache-commons-jexl_commons-jexl.jar -> /usr/share/java/apache-commons-jexl/commons-jexl.jar
tomcat tomcat    44 Apr 19 19:34 apache-commons-validator.jar -> /usr/share/java/apache-commons-validator.jar
tomcat tomcat    24 Apr 19 19:34 bcel.jar -> /usr/share/java/bcel.jar
tomcat tomcat    41 Apr 19 19:34 byte-buddy_byte-buddy.jar -> /usr/share/java/byte-buddy/byte-buddy.jar
tomcat tomcat    24 Apr 19 19:34 c3p0.jar -> /usr/share/java/c3p0.jar
tomcat tomcat    31 Apr 19 19:34 cglib_cglib.jar -> /usr/share/java/cglib/cglib.jar
tomcat tomcat    39 Apr 19 19:34 classmate_classmate.jar -> /usr/share/java/classmate/classmate.jar
tomcat tomcat    37 Apr 19 19:34 commons-beanutils.jar -> /usr/share/java/commons-beanutils.jar
tomcat tomcat    31 Apr 19 19:34 commons-cli.jar -> /usr/share/java/commons-cli.jar
tomcat tomcat    33 Apr 19 19:34 commons-codec.jar -> /usr/share/java/commons-codec.jar
tomcat tomcat    39 Apr 19 19:34 commons-collections.jar -> /usr/share/java/commons-collections.jar
tomcat tomcat    36 Apr 19 19:34 commons-digester.jar -> /usr/share/java/commons-digester.jar
tomcat tomcat    37 Apr 19 19:34 commons-discovery.jar -> /usr/share/java/commons-discovery.jar
tomcat tomcat    30 Apr 19 19:34 commons-el.jar -> /usr/share/java/commons-el.jar
tomcat tomcat    38 Apr 19 19:34 commons-fileupload.jar -> /usr/share/java/commons-fileupload.jar
tomcat tomcat    30 Apr 19 19:34 commons-io.jar -> /usr/share/java/commons-io.jar
tomcat tomcat    33 Apr 19 19:34 commons-lang3.jar -> /usr/share/java/commons-lang3.jar
tomcat tomcat    35 Apr 19 19:34 commons-logging.jar -> /usr/share/java/commons-logging.jar
tomcat tomcat    30 Apr 19 19:34 concurrent.jar -> /usr/share/java/concurrent.jar
tomcat tomcat    75 Apr 19 19:34 concurrentlinkedhashmap-lruconcurrentlinkedhashmap-lru.jar -> /usr/share/java/concurrentlinkedhashmap-lru/concurrentlinkedhashmap-lru.jar
tomcat tomcat    25 Apr 19 19:34 dom4j.jar -> /usr/share/java/dom4j.jar
tomcat tomcat    23 Apr 19 19:34 dwr.jar -> /usr/share/java/dwr.jar
tomcat tomcat    32 Apr 19 19:34 ehcache-core.jar -> /usr/share/java/ehcache-core.jar
tomcat tomcat    40 Apr 19 19:34 geronimo-jta-1.1-api.jar -> /usr/share/java/geronimo-jta-1.1-api.jar
tomcat tomcat    44 Apr 19 19:34 glassfish-activation-api.jar -> /usr/share/java/glassfish-activation-api.jar
tomcat tomcat    47 Apr 19 19:34 glassfish-jaxb_jaxb-runtime.jar -> /usr/share/java/glassfish-jaxb/jaxb-runtime.jar
tomcat tomcat    39 Apr 19 19:34 glassfish-jaxb_txw2.jar -> /usr/share/java/glassfish-jaxb/txw2.jar
tomcat tomcat    43 Apr 19 19:34 google-gson_google-gsongson-extras.jar -> /usr/share/java/google-gson/gson-extras.jar
tomcat tomcat    36 Apr 19 19:34 google-gson_google-gsongson.jar -> /usr/share/java/google-gson/gson.jar
tomcat tomcat    79 Apr 19 19:34 hibernate-commons-annotations_hibernate-commons-annotations.jar -> /usr/share/java/hibernate-commons-annotations/hibernate-commons-annotations.jar
tomcat tomcat    54 Apr 19 19:34 hibernate-types_hibernate-types-52.jar -> /usr/share/java/hibernate-types/hibernate-types-52.jar
tomcat tomcat    45 Apr 19 19:34 hibernate5_hibernate-c3p0.jar -> /usr/share/java/hibernate5/hibernate-c3p0.jar
tomcat tomcat    45 Apr 19 19:34 hibernate5_hibernate-core.jar -> /usr/share/java/hibernate5/hibernate-core.jar
tomcat tomcat    48 Apr 19 19:34 hibernate5_hibernate-ehcache.jar -> /usr/share/java/hibernate5/hibernate-ehcache.jar
tomcat tomcat    35 Apr 19 19:34 httpasyncclient.jar -> /usr/share/java/httpasyncclient.jar
tomcat tomcat    45 Apr 19 19:34 httpcomponents_httpclient.jar -> /usr/share/java/httpcomponents/httpclient.jar
tomcat tomcat    47 Apr 19 19:34 httpcomponents_httpcore-nio.jar -> /usr/share/java/httpcomponents/httpcore-nio.jar
tomcat tomcat    43 Apr 19 19:34 httpcomponents_httpcore.jar -> /usr/share/java/httpcomponents/httpcore.jar
tomcat tomcat    26 Apr 19 19:34 ical4j.jar -> /usr/share/java/ical4j.jar
tomcat tomcat    42 Apr 19 19:34 istack-commons-runtime.jar -> /usr/share/java/istack-commons-runtime.jar
tomcat tomcat    39 Apr 19 19:34 jackson-annotations.jar -> /usr/share/java/jackson-annotations.jar
tomcat tomcat    32 Apr 19 19:34 jackson-core.jar -> /usr/share/java/jackson-core.jar
tomcat tomcat    36 Apr 19 19:34 jackson-databind.jar -> /usr/share/java/jackson-databind.jar
tomcat tomcat    33 Apr 19 19:34 jade4jjade4j.jar -> /usr/share/java/jade4j/jade4j.jar
root   root      36 Sep 28  2022 java-branding.jar -> /usr/share/rhn/lib/java-branding.jar
tomcat tomcat    34 Apr 19 19:34 java-saml-core.jar -> /usr/share/java/java-saml-core.jar
tomcat tomcat    29 Apr 19 19:34 java-saml.jar -> /usr/share/java/java-saml.jar
tomcat tomcat    29 Apr 19 19:34 javassist.jar -> /usr/share/java/javassist.jar
tomcat tomcat    30 Apr 19 19:34 javax.mail.jar -> /usr/share/java/javax.mail.jar
tomcat tomcat    47 Apr 19 19:34 jboss-logging.jar -> /usr/share/java/jboss-logging/jboss-logging.jar
tomcat tomcat    27 Apr 19 19:34 jcommon.jar -> /usr/share/java/jcommon.jar
tomcat tomcat    40 Apr 19 19:34 jctools_jctools-core.jar -> /usr/share/java/jctools/jctools-core.jar
tomcat tomcat    24 Apr 19 19:34 jdom.jar -> /usr/share/java/jdom.jar
tomcat tomcat    29 Apr 19 19:34 joda-time.jar -> /usr/share/java/joda-time.jar
tomcat tomcat    26 Apr 19 19:34 jose4j.jar -> /usr/share/java/jose4j.jar
tomcat tomcat    51 Apr 19 19:34 jpa-api_jakarta.persistence-api.jar -> /usr/share/java/jpa-api/jakarta.persistence-api.jar
tomcat tomcat    35 Apr 19 19:34 log4j_log4j-api.jar -> /usr/share/java/log4j/log4j-api.jar
tomcat tomcat    36 Apr 19 19:34 log4j_log4j-core.jar -> /usr/share/java/log4j/log4j-core.jar
tomcat tomcat    42 Apr 19 19:34 log4j_log4j-slf4j-impl.jar -> /usr/share/java/log4j/log4j-slf4j-impl.jar
tomcat tomcat    56 Apr 19 19:34 mchange-commons_mchange-commons-java.jar -> /usr/share/java/mchange-commons/mchange-commons-java.jar
tomcat tomcat    38 Apr 19 19:34 netty_netty-buffer.jar -> /usr/share/java/netty/netty-buffer.jar
tomcat tomcat    37 Apr 19 19:34 netty_netty-codec.jar -> /usr/share/java/netty/netty-codec.jar
tomcat tomcat    38 Apr 19 19:34 netty_netty-common.jar -> /usr/share/java/netty/netty-common.jar
tomcat tomcat    39 Apr 19 19:34 netty_netty-handler.jar -> /usr/share/java/netty/netty-handler.jar
tomcat tomcat    40 Apr 19 19:34 netty_netty-resolver.jar -> /usr/share/java/netty/netty-resolver.jar
tomcat tomcat    60 Apr 19 19:34 netty_netty-transport-native-unix-common.jar -> /usr/lib64/java/netty/netty-transport-native-unix-common.jar
tomcat tomcat    41 Apr 19 19:34 netty_netty-transport.jar -> /usr/share/java/netty/netty-transport.jar
tomcat tomcat    37 Apr 19 19:34 objectweb-asm_asm.jar -> /usr/share/java/objectweb-asm/asm.jar
root   root      39 Apr 19 19:34 ongres-scram_client.jar -> /usr/share/java/ongres-scram/client.jar
root   root      39 Apr 19 19:34 ongres-scram_common.jar -> /usr/share/java/ongres-scram/common.jar
root   root      46 Apr 19 19:34 ongres-stringprep_saslprep.jar -> /usr/share/java/ongres-stringprep/saslprep.jar
root   root      48 Apr 19 19:34 ongres-stringprep_stringprep.jar -> /usr/share/java/ongres-stringprep/stringprep.jar
tomcat tomcat    23 Apr 19 19:34 oro.jar -> /usr/share/java/oro.jar
tomcat tomcat    39 Apr 19 19:34 pgjdbc-ng_pgjdbc-ng.jar -> /usr/share/java/pgjdbc-ng/pgjdbc-ng.jar
tomcat tomcat    33 Apr 19 19:34 pgjdbc-ng_spy.jar -> /usr/share/java/pgjdbc-ng/spy.jar
root   root      35 Apr 19 19:34 postgresql-jdbc.jar -> /usr/share/java/postgresql-jdbc.jar
tomcat tomcat    26 Apr 19 19:34 quartz.jar -> /usr/share/java/quartz.jar
tomcat tomcat    42 Apr 19 19:34 redstone-xmlrpc-client.jar -> /usr/share/java/redstone-xmlrpc-client.jar
tomcat tomcat    35 Apr 19 19:34 redstone-xmlrpc.jar -> /usr/share/java/redstone-xmlrpc.jar
tomcat tomcat    26 Apr 19 19:34 rhn.jar -> /usr/share/rhn/lib/rhn.jar
tomcat tomcat    57 Apr 19 19:34 salt-netapi-clientsalt-netapi-client.jar -> /usr/share/java/salt-netapi-client/salt-netapi-client.jar
tomcat tomcat    31 Apr 19 19:34 simple-core.jar -> /usr/share/java/simple-core.jar
tomcat tomcat    30 Apr 19 19:34 simple-xml.jar -> /usr/share/java/simple-xml.jar
tomcat tomcat    32 Apr 19 19:34 simpleclient.jar -> /usr/share/java/simpleclient.jar
tomcat tomcat    39 Apr 19 19:34 simpleclient_common.jar -> /usr/share/java/simpleclient_common.jar
tomcat tomcat    42 Apr 19 19:34 simpleclient_hibernate.jar -> /usr/share/java/simpleclient_hibernate.jar
tomcat tomcat    43 Apr 19 19:34 simpleclient_httpserver.jar -> /usr/share/java/simpleclient_httpserver.jar
tomcat tomcat    40 Apr 19 19:34 simpleclient_servlet.jar -> /usr/share/java/simpleclient_servlet.jar
tomcat tomcat    28 Apr 19 19:34 sitemesh.jar -> /usr/share/java/sitemesh.jar
tomcat tomcat    29 Apr 19 19:34 slf4j_api.jar -> /usr/share/java/slf4j/api.jar
tomcat tomcat    29 Apr 19 19:34 snakeyaml.jar -> /usr/share/java/snakeyaml.jar
tomcat tomcat    41 Apr 19 19:34 spark-corespark-core.jar -> /usr/share/java/spark-core/spark-core.jar
tomcat tomcat    59 Apr 19 19:34 spark-template-jadespark-template-jade.jar -> /usr/share/java/spark-template-jade/spark-template-jade.jar
tomcat tomcat    30 Apr 19 19:34 statistics.jar -> /usr/share/java/statistics.jar
tomcat tomcat    28 Apr 19 19:34 stax-api.jar -> /usr/share/java/stax-api.jar
tomcat tomcat    29 Apr 19 19:34 stax2-api.jar -> /usr/share/java/stax2-api.jar
tomcat tomcat    35 Apr 19 19:34 stringtree-json.jar -> /usr/share/java/stringtree-json.jar
tomcat tomcat    26 Apr 19 19:34 struts.jar -> /usr/share/java/struts.jar
tomcat tomcat    41 Apr 19 19:34 taglibs-standard-impl.jar -> /usr/share/java/taglibs-standard-impl.jar
tomcat tomcat    43 Apr 19 19:34 taglibs-standard-jstlel.jar -> /usr/share/java/taglibs-standard-jstlel.jar
tomcat tomcat    41 Apr 19 19:34 taglibs-standard-spec.jar -> /usr/share/java/taglibs-standard-spec.jar
tomcat tomcat    37 Apr 19 19:34 woodstox-core-asl.jar -> /usr/share/java/woodstox-core-asl.jar
tomcat tomcat    39 Apr 19 19:34 xalan-j2-serializer.jar -> /usr/share/java/xalan-j2-serializer.jar
tomcat tomcat    28 Apr 19 19:34 xalan-j2.jar -> /usr/share/java/xalan-j2.jar
tomcat tomcat    29 Apr 19 19:34 xerces-j2.jar -> /usr/share/java/xerces-j2.jar
tomcat tomcat    26 Apr 19 19:34 xmlsec.jar -> /usr/share/java/xmlsec.jar
```

**Comment by juliogonzalez** [comment]
@j-kihet 

> Previous report https://github.com/uyuni-project/uyuni/issues/6552
> ls -la /srv/tomcat/webapps/rhn/WEB-INF/lib/

As we can't see colors... are any of those link broken? In a console with color such links should show in red, while other links should be in cyan.

**Comment by mcalmer** [comment]
`NoClassDefFoundError: Could not initialize class com.redhat.rhn.domain.notification.UserNotificationFactory`

This class is compiled in rhn.jar.
No idea how you made this error. Only when you tried to compile rhn.jar on your own.

What does:
```
$> rpm -qf /usr/share/rhn/lib/rhn.jar
```
return?

**Comment by j-kihet** [comment]
Everything is updated from the official repo.
```
rpm -qf /usr/share/rhn/lib/rhn.jar
spacewalk-java-lib-4.4.17-220400.1.4.uyuni2.noarch
```

**Comment by stdevel** [comment]
I have the same issues with SUSE Manager 4.3 and already raised a support case.

Do you have any news in addition?

**Comment by mikeletux** [comment]
@j-kihet could you please let us know which version you upgrade it from?
I've deployed uyuni 2023.04 in a local environment and the notification page works for me:
![image](https://github.com/uyuni-project/uyuni/assets/6768988/d45ab291-8ae0-44af-90a8-ecde0213cc2f)

@stdevel could you send me through email the support case so I can try to correlate with this one?
Thanks!

**Comment by j-kihet** [comment]
> @j-kihet could you please let us know which version you upgrade it from? I've deployed uyuni 2023.04 in a local environment and the notification page works for me: ![image](https://user-images.githubusercontent.com/6768988/267055802-d45ab291-8ae0-44af-90a8-ecde0213cc2f.png)
> 
> @stdevel could you send me through email the support case so I can try to correlate with this one? Thanks!

Hi , im stuck on Uyuni release 2022.11. Everything after that ends up with the same issue. I have tried to update to every release after 2022.11.

**Comment by stdevel** [comment]
Thanks for pointing out - that is good to know. I have a 2023.04 system which has no problem with the notification center. So this looks like a bug then?

**Comment by j-kihet** [comment]
Seems like a bug, every other feature/system works.

**Comment by j-kihet** [comment]
Fixed in 2023.09!
Thank you everyone!
